web/public/js/controllers/user-profile-controller.js

```diff
@@ -173,6 +173,16 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
     return cdProgrammingLanguagesService.get();
   };
 
+
+  $scope.imageByLanguage = function(language){
+    cdProgrammingLanguagesService.getByLanguage(language,
+      function (responseData) {
+        $scope.profile.programmingLanguagesImageUrl.push(responseData[0]);
+    }, function () {
+      console.error("error when get data")
```

Comment:
That's I see.. I can use [$log](https://docs.angularjs.org/api/ng/service/$log) if you want?